ui/src/app/admin/services/AdminService.js

```diff
@@ -8,15 +8,15 @@
     var startDownloads = function() {
       return $http({
           method: 'GET',
-          url: BassaUrl + '/download/start',
+          url: BassaUrl + '/api/download/start',
```

Comment:
Does we have backend changed for it as well? @kmehant Any idea?

Reply:
Because I guess `/api/` will be appended to actual request from ingress as well?

Reply:
@vivonk 
We have the server routes in `/api/` form and two routes belonging to start and kill downloads were also changed to that form. Server endpoints now all are in `/api/` form so in the same sense I have changed them in client-side too.

In the case of ingress the host is the same so had to add extension `/api/` to differentiate between client and server service.